### ISSUE TITLE
Example article's 

### ISSUE THREAD

**Issue by josephreed2600** [AndyClifton/strucPDF#6]
**Describe the bug**
Text objects in the example article are not tagged, resulting in a failed accessibility check.

**To Reproduce**
Steps to reproduce the behaviour:
1. Obtain the free [PDF Accessibility Checker](https://www.access-for-all.ch/en/pdf-lab/pdf-accessibility-checker-pac/downloading-pac.html) from Zugang für alle. This issue was checked with PAC 3.
2. Open the example article PDF in the checker.
3. Below the listing of checkpoints, select "Results in Detail" to see a breakdown of the untagged objects.

**Expected behavior**
Text objects in a PDF compiled with this package should probably be tagged; this feels to me within the scope of the project.

**Log messages**
[article_PAC_Report.pdf](https://github.com/AndyClifton/strucPDF/files/3517531/article_PAC_Report.pdf)
![TextObjectNotTagged](https://user-images.githubusercontent.com/17363676/63296715-9a32c180-c284-11e9-83ef-1d4be58f49a4.PNG)

**Additional notes**
This is not isolated to the example article; I discovered this while working on my own document.

I appreciate the work you've put into this project! If there's anything I can do to help, I'd be glad to. I'm by no means an expert on LaTeX or PDF, but I've been studying the incantations lately for my accessibility projects, and I'm quite eager to see this work as well as it can. I'd rather spend a week in vi writing LaTeX than an hour in Acrobat.
